nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -72,6 +72,11 @@ BtNavigator::BtNavigator()
   declare_parameter("global_frame", std::string("map"));
   declare_parameter("robot_base_frame", std::string("base_link"));
   declare_parameter("odom_topic", std::string("odom"));
+  declare_parameter("enable_groot_monitoring", false);
```

Comment:
Maybe worth dropping the `nt_` prefix? I'm not sure what it adds or that people will understand what that means.

Maybe instead do `pose_bt_...` and `poses_bt_...`?